### PR TITLE
fix: aaed audit fix and network config

### PR DESCRIPTION
**Pull request by AlinaDiadickina**

### CODE REVIEW

File: packages/smart-contracts/packages/vesting/contracts/Vesting.sol [hunk 1]
```diff
@@ -240,17 +240,14 @@ contract Vesting is IVesting, Initializable, AccessControlUpgradeable {
...
 
+        // Effects: Increase locked pool token amount
+        pool.dedicatedPoolTokenAmount += tokenAmount;
+
         // Checks: User token amount should not exceed total pool amount
-        if (
-            pool.totalPoolTokenAmount <
-            (pool.dedicatedPoolTokenAmount + tokenAmount)
-        ) {
+        if (pool.totalPoolTokenAmount < (pool.dedicatedPoolTokenAmount)) {
```
Comment (BenasVolkovas) [inline packages/smart-contracts/packages/vesting/contracts/Vesting.sol:247]: `(pool.dedicatedPoolTokenAmount)` -> `pool.dedicatedPoolTokenAmount`

File: packages/smart-contracts/packages/nft/scripts/data/networkConfig.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
         "nft_symbol": "WOWPASS",
         "vesting_contract": "",
         "vesting_pool_id": 0,
-        "max_level": 5,
+        "max_level": 10,
```
Comment (BenasVolkovas) [inline packages/smart-contracts/packages/nft/scripts/data/networkConfig.json:42]: Why is this changing? Never heard that we will have 10 nft levels

**Review by BenasVolkovas — CHANGES_REQUESTED** [review]

File: packages/smart-contracts/packages/nft/scripts/data/networkConfig.json (hunk 2 above)
Comment (AlinaDiadickina) [reply packages/smart-contracts/packages/nft/scripts/data/networkConfig.json:42]: but we are adding genesis levels, thus it goes up (genesis level id 6 = level 1, genesis id 7 = level 2)

Comment (BenasVolkovas) [reply packages/smart-contracts/packages/nft/scripts/data/networkConfig.json:42]: But it doesn't work like that, we only have 5 level all the time

**Review by BenasVolkovas — APPROVED** [review]

**Pull request merged**